packages/nextjs/tailwind.config.js

```diff
@@ -30,6 +30,14 @@ module.exports = {
           ".tooltip": {
             "--tooltip-tail": "6px",
           },
+          ".nested-tooltip": {
+            visibility: "hidden",
+            position: "absolute",
+          },
+          ".has-tooltip:hover .nested-tooltip": {
+            visibility: "visible",
+            "z-index": "100",
+          },
```

Comment:
Why do we need a custom tooltip?

We always keep a minimal approach and try to use what we already have (if possible, ofc!) If not, adding this kind of thing on PRs will result in a super bloated codebase.

Reply:
I wanted it for two reasons, but I'm not 100% it's needed yet.
1. To make it possible for the dev to copy from the tooltip.
2. To change layout so that the tooltip does not break lines for the contract path. But perhaps that's overkill?

Reply:
Maybe «overkill» is too strong of a word haha but since SE2 is a starter kit we always want to keep things as minimal & simple as possible so we don't end with a bunch of stuff.

So I'd say: let's use the existing tooltip, and we can always iterate in the future.

Thanks!

Reply:
If we are using the exact same thing 3 times, it's probably a good idea to have a separate component.

Reply:
I changed it in to one component and went back to normal tooltip.

https://github.com/scaffold-eth/scaffold-eth-2/assets/6756284/e0af8aff-3a06-4bdc-af47-30ae1fb97efc